packages/plugins/rubygem-gapic-common/rubygem-gapic-common.spec

```diff
@@ -0,0 +1,66 @@
+# template: default
+%global gem_name gapic-common
+
+Name: rubygem-%{gem_name}
+Version: 0.12.0
+Release: 1%{?dist}
+Summary: Common code for GAPIC-generated API clients
+License: Apache-2.0
+URL: https://github.com/googleapis/gapic-generator-ruby
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: ruby >= 2.6
+BuildRequires: ruby >= 2.6
+BuildRequires: rubygems-devel
+BuildArch: noarch
+# end specfile generated dependencies
+
+%description
+Common code for GAPIC-generated API clients.
+
+
+%package doc
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -q -n  %{gem_name}-%{version}
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build ../%{gem_name}-%{version}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
```

Comment:
imho we can
```
%exclude %{gem_instdir}/etc
%exclude %{gem_instdir}/include
%exclude %{gem_instdir}/third_party
```

as they don't seem to contain files that are needed at runtume (not sure about etc, actually)